.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,63 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [Improvement, Bug, Other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Which packages do this issue concern?
+      options: [NPM, Packagist]
+      multiple: true
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Simple Icons Version
+      placeholder: "Example: 5.11.0"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Other Software
+      description: >-
+        Software with versions if known (Node.js & NPM, PHP & Packagist,
+        Browser). If you are unsure, enter "None."
+      placeholder: |
+        Example:
+        Node.js v16
+        Chrome
```

This example should probably also include the version, don't have chrome on this system (and I'm not sure if Chromium is at the same version), but for Firefox it would be:

```suggestion
        Firefox 91.0.1 (64-bit)
```

(I'd prefer the example to be for Chrome, as it's more commonly used)